content/doc/book/pipeline/syntax.adoc

```diff
@@ -573,8 +573,8 @@ link:https://issues.jenkins-ci.org/browse/INFRA-1053[INFRA-1503].
 The `triggers` directive defines the automated ways in which the Pipeline
 should be re-triggered. For Pipelines which are integrated with a source such
 as GitHub or BitBucket, `triggers` may not be necessary as webhooks-based
-integration will likely already be present. Currently the only two available
-triggers are `cron` and `pollSCM`.
+integration will likely already be present. Currently the only available
+triggers are `cron`, `pollSCM` and `upstream`.
```

Comment:
"the only available" is going to become wrong once any new trigger is introduced. Ideally requires rephrasing, CC @gilesgas

Reply:
Thanks @oleg-nenashev - in fact you can do away with the word "only" (which sounds slightly negative) by rewording this sentence to:
"The triggers which are currently available are `cron`, `pollSCM` and `upstream`."
Then it doesn't matter how many more triggers you add in future, you'll no longer have to worry about rewording the first part of this sentence.

Reply:
This suggested is implemented in [PR 1206](https://github.com/jenkins-infra/jenkins.io/pull/1206).